bin/Kernelize/Kernelize.cpp

```diff
@@ -19,6 +19,17 @@ static llvm::cl::extrahelp
 static llvm::cl::extrahelp
     g_more_help("\nAnalyzes Linux kernel RCU macro usage\n");
 
+static llvm::cl::opt<bool> g_print_locations(
+    "locations",
```

Comment:
Maybe abstract this name into a variable since the description of `g_print_locations_alias` references it as well?

Reply:
🤷 here I wouldn't be bothered by it much as it is a "config" in a sense - if the referencing isn't provided by the cli framework then this is good enough.